### ISSUE TITLE
Unexpected behavior for `import Fragment from react`

### ISSUE THREAD

**Issue by sequencerr** [sweepline/eslint-plugin-unused-imports#61]
`import { Fragment } from "react";`
the Fragment isn't removed. For `import React, { Fragment } from "react";` same behavior 

I'm notified only by typescript validator:
`'Fragment' is declared but its value is never read. ts(6133)`

**Comment by sweepline** [comment]
It sounds like a configuration error, can you please post that too?

**Comment by sequencerr** [comment]
@sweepline Probably related to https://github.com/sweepline/eslint-plugin-unused-imports/issues/63
I guess it was really config problem for Fragment. Not actual problem for me, project I worked for had complicated structure.

Maybe I don't understand something but why React not removed in 
```js
import React, { Fragment } from "react";

console.log(<></>);
console.log(<div></div>);
```
![image](https://github.com/sweepline/eslint-plugin-unused-imports/assets/45060278/1f94d37b-b52f-453a-ac11-0b6ff3239c35)

As far as I know, bundlers automatically prepend jsx-runtime for `.jsx`/`.tsx` files. Probably not your plugin problem, but as you mentioned, the eslint's `no-unused-vars`

**Comment by sequencerr** [comment]
Here is repro repo https://github.com/sequencerr/eslint-plugin-unused-imports-issue-repro
